cephfs/file_test.go

```diff
@@ -769,3 +769,121 @@ func TestSync(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestFilePreadvPwritev(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	fname := "TestFilePreadvPwritev.txt"
+	defer mount.Unlink(fname)
+
+	t.Run("simple", func(t *testing.T) {
+		f, err := mount.Open(fname, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)
```

Comment:
nit: recommend to default to sane permissions, like 0644. It could prevent security issues when otehrs copy/paste  from the tests.

Reply:
Heh. OK, sure.

Reply:
Done. Along with older test  funcs in a separate commit.